src/libraries/System.Private.Xml/src/System/Xml/Serialization/SchemaImporter.cs

```diff
@@ -13,6 +13,7 @@
 
 #if DEBUG
     using System.Diagnostics;
+    using System.Diagnostics.CodeAnalysis;
```

Comment:
You are using this outside of `#if DEBUG` below. Shouldn't this be in the group above?